src/main/java/seedu/duke/atomichabit/atomichabit/AtomicHabit.java

```diff
@@ -0,0 +1,26 @@
+package seedu.duke.atomichabit.atomichabit;
+
+public class AtomicHabit {
+    protected String description;
+
+    protected int count;
+
+    public AtomicHabit(String description) {
+        this.description = description;
+        count = 0;
```

Comment:
Would be good to standardise how you access the class attributes in the constructor, i.e. `this.count` instead of just `count`